R/R5.R

```diff
@@ -48,7 +48,7 @@
 #' ## Exporting to PDF (default)
 #' myReport$export()
 #'
-#' ## Or to docx in tempdir():
+#' # Or to docx in tempdir():
```

Comment:
@RomanTsegelskyi I think we consistently & always used double hashes for standalone comments on their own and used single hash for inline comments (after actual R code). I'd be happy to keep that standard :)